arviz/data/inference_data.py

```diff
@@ -339,7 +319,7 @@ def to_json(self, filename, **kwargs):
         str
             Location of json file
         """
-        idata_dict = self.to_dict()
+        idata_dict = _make_json_serializable(self.to_dict())
 
         with open(filename, "w") as file:
```

Comment:
would "wb" work here?

Reply:
It's returning the error `TypeError: a bytes-like object is required, not 'str'`.